test/unit/services/url.ts

```diff
@@ -146,10 +138,54 @@ suite('URL Service', ({ expect, spy, stub }) => {
         }
       };
 
-      service.augmentHistory();
+      service.augmentHistory('', {});
 
       expect(once).to.be.calledWith(Events.HISTORY_SAVE, service.listenForHistoryChange);
-      expect(dispatch).to.be.calledWith(fetchProductsAction);
+    });
+
+    it('should request products', () => {
+      const title = 'Search Page';
+      const once = stub();
```

Comment:
does `once` have to be a `stub()` here? It doesn't look like we make any expectations using it